src/Avalonia.Controls/TextBox.cs

```diff
@@ -1684,7 +1714,7 @@ protected override void OnPointerReleased(PointerReleasedEventArgs e)
 
                 if(SelectionStart != SelectionEnd)
                 {
-                    RaiseEvent(new ContextRequestedEventArgs(e));
+                    _presenter.Canvas?.ShowContextMenu();
```

Comment:
Doesn't this change prevent the user from customizing/handling the context menu manually?

Reply:
It now internally calls `ContextRequestedEventArgs`.